sparkles/tests/test_checks.py

```diff
@@ -144,25 +138,22 @@ def test_check_jupiter_distribution_fail(aca_review_table):
         **mod_std_info(detector="HRC-I"), duration=20000, stars=stars, dark=DARK40
     )
     acar = aca_review_table(aca)
-    acar.jupiter = Table([{"time": CxoTime(acar.date).secs, "row": 300, "col": 10}])
-    check_jupiter_distribution(acar)
-    assert acar.messages == [
-        {
-            "category": "critical",
-            "text": "Jupiter guide star distribution check failed. Need 2 guide stars always opposite Jupiter.",
-        }
-    ]
-
-
-@pytest.mark.parametrize("aca_review_table", (ACAReviewTable, ACACheckTable))
-def test_check_jupiter_distribution_fine(aca_review_table):
-    stars = StarsTable.empty()
-    stars.add_fake_constellation(n_stars=3, mag=8.5)
-    aca = get_aca_catalog(**STD_INFO, duration=20000, stars=stars, dark=DARK40)
-    acar = aca_review_table(aca)
-    acar.jupiter = Table([{"time": CxoTime(acar.date).secs, "row": -300, "col": 10}])
+    acar.jupiter = Table(
+        [{"time": CxoTime(acar.date).secs, "row": jupiter_row, "col": 10}]
+    )
     check_jupiter_distribution(acar)
-    assert acar.messages == []
+    exp = (
+        []
+        if jupiter_row < 0
```

Comment:
The true check has padding, but if we update this test to use different parameters, hopefully we'll be able to figure it out.